### PR TITLE
feat: database + routing

### PR DESCRIPTION
**Pull request by rkrishn7**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

Adds database migration config, database client + express.

- **What is the current behavior?** (You can also link to an open issue here)

- **What is the new behavior (if this is a feature change)?**

- **Other information**:

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  url      = env("DATABASE_URL")
+}
+
+model migrations {
+  id     Int      @id @default(autoincrement())
+  name   String
+  run_on DateTime
+}
+
+model parties {
```
Comment (RyanMatthewJacobs) [inline prisma/schema.prisma:16]: Are we going to add a `user` model as well?

Comment (rkrishn7) [reply prisma/schema.prisma:16]: yep, will do that. Just need to be sure about the data model

File: src/lib/socket/server.ts [hunk 2]
```diff
@@ -45,4 +48,22 @@ export default class Manager {
       })
     );
   }
+
+  async onSocketSendMessage(this: io.Socket, data: any) {
+    await MessageService.create(data);
+    this.broadcast.emit('new_message', { message: data });
```
Comment (RyanMatthewJacobs) [inline src/lib/socket/server.ts]: Won't broadcast.emit() send this message to all clients connected to the server, not just those in the user's party. I might be wrong on this, but we will only want to send messages between users in the same party.

**Review by RyanMatthewJacobs — DISMISSED** [review]
Looks good! Code is formatted nicely. We'll want to start creating unit tests for all of our service and controller classes, but that can be done later.

**Comment by RyanMatthewJacobs** [comment]
Can we merge this branch now @rkrishn7?

**Review by zzulanas — DISMISSED** [review]

**Pull request merged**